app/assets/javascripts/relationships.js.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Can you remove this and other empty files? We aren't currently using any .coffee or .scss files and these are just templates in any case. Thanks!